manifests/resource/vhost.pp

```diff
@@ -206,6 +210,8 @@
   $ssl_stapling_responder       = undef,
   $ssl_stapling_verify          = false,
   $ssl_session_timeout          = '5m',
+  $ssl_session_tickets          = 'on',
```

Comment:
This should default undef, and be conditional in the template.

Since it was introduced in 1.5.9, and some distros support older versions than that, this directive would be unrecognized by nginx and prevent it from starting.